src/algorithms/tracking/TrackParamTruthInit.cc

```diff
@@ -75,7 +75,7 @@ eicrecon::TrackParamTruthInit::produce(const edm4hep::MCParticleCollection* mcpa
     const auto pdg      = mcparticle.getPDG();
     const auto particle = m_particleSvc.particle(pdg);
     double charge       = std::copysign(1.0, particle.charge);
-    if (abs(charge) < std::numeric_limits<double>::epsilon()) {
+    if (std::abs(charge) < std::numeric_limits<double>::epsilon()) {
```

Comment:
Can I confirm that this is a bug which should be addressed in a different PR.

Reply:
https://github.com/eic/EICrecon/pull/1878